File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    x += "";
    let pL = 0;
    let pR = x.length - 1;
    while(pL<pR){
        if(x.charAt(pL) != x.charAt(pR)){
            return false;
        }else{
            pL++;
            pR--;
        }
    }
    return true;
};

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        String num = Integer.toString(x);
        int pL = 0;
        int pR = num.length()-1;
        while(pL<pR){
            if(num.charAt(pL) != num.charAt(pR)){
                return false;
            }else{
                pL++;
                pR--;
            }
        }
        return true;
    }
}